Drop-In/src/sharedTest/java/com.braintreepayments.api.test/ExpirationDate.java

```diff
@@ -2,5 +2,5 @@
 
 public class ExpirationDate {
 
-    public static final String VALID_EXPIRATION = "1219";
+    public static final String VALID_EXPIRATION = "1229";
```

Comment:
This is far in the future, but it would probably be best to programmatically generate a date based on the current year.

Reply:
+1. We do this it dynamically in the base SDK 
https://github.braintreeps.com/team-sdk/braintree-android/blob/master/TestUtils/src/main/java/com/braintreepayments/testutils/ExpirationDateHelper.java

Reply:
Yeah, I had considered that.  It would be a bit more work since that const is used quite a few places, but if we think it's worth doing here, I'm happy to do some reworking there, too.  I gambled originally on getting the tests working, and potentially adding tasks for improvements, given these have been failing since ~December. 😱

Reply:
That's also fine! Creating a ticket for it works 👍